.pylintrc

```diff
@@ -317,7 +317,7 @@ max-args=5
 ignored-argument-names=_.*
 
 # Maximum number of locals for function / method body
-max-locals=15
+max-locals=20
```

Comment:
Why do we need to change this rule globally?
Are we able to disable the rule as an special case for the offending block instead?

Reply:
Good callout, let me change this for individual functions rather than the whole project